src/main/scala/uk/gov/nationalarchives/consignmentexport/BagMetadata.scala

```diff
@@ -23,15 +23,8 @@ class BagMetadata(keycloakClient: KeycloakClient)(implicit val logger: SelfAware
   }
 
   private def getConsignmentDetails(consignment: GetConsignment, exportDatetime: ZonedDateTime): Map[String, Option[String]] = {
-    val seriesCode = for {
-      series <- consignment.series
-      sc <- series.code
-    } yield sc
-
-    val bodyName = for {
-      body <- consignment.transferringBody
-      bc <- body.name
-    } yield bc
+    val seriesCode = for (series <- consignment.series) yield series.code
```

Comment:
Do you still need the `for` comprehensions here? They're normally only useful when you have multiple calls to `map` or `flatmap`.

Reply:
🙂  This was what I was thinking, I kept it to stay consistent, but really and truly, we can just use maps for everything up to like 34.